src/main/java/executor/command/CommandBlank.java

```diff
@@ -0,0 +1,24 @@
+package executor.command;
+
+import executor.task.TaskList;
+import ui.Ui;
+import ui.Wallet;
+
+public class CommandBlank extends Command {
+    // Constructor
+    public CommandBlank() {
+    }
+
+    /**
+     * Executes a particular Command.
+     */
+    @Override
+    public void execute(TaskList taskList) {
+        Ui.printSeparator();
+    }
+
+    @Override
+    public void execute(Wallet wallet) {
+
```

Comment:
Why is this empty? Empty functions are pointless. Why not just skip the override here and let it do the default?